packages/frontend/src/components/Election/Results/Pages.tsx

```diff
@@ -0,0 +1,24 @@
+import { Pagination } from "@mui/material";
+import Widget from "./components/Widget";
+import WidgetContainer from "./components/WidgetContainer";
+
+const Pages = ({
+  children, pageCount, title, page, setPage /* one-based */
```

Comment:
Good comment! This has tripped me up before when working on this

I also like that you've Pages abstracted for reuse in other areas